app/views/tags/index.html.erb

```diff
@@ -19,25 +19,52 @@
   <meta name="twitter:image:src" content="<%= SiteConfig.main_social_image %>">
 <% end %>
 
-<div class="user-profile-header tag-header bg-transparent">
-  <div class="tag-or-query-header-container">
+<style>
+  .tag-card:not(:hover) {
+    box-shadow: 0 0 0 1px var(--card-border) !important;
+    border: 2px solid transparent !important;
+  }
+
+  .tag-card:not(:hover) .crayons-btn--secondary.follow-action-button {
+    background: var(--button-secondary-bg) !important;
+    color: var(--button-secodnary-color) !important;
+  }
+</style>
+
+<div class="crayons-layout crayons-layout--1-col">
+  <header class="px-2 m:px-0">
     <h1>Top 100 Tags</h1>
```

Comment:
Just thought that there's no guarantee that a website has 100 tags :D 
https://letsbuild.gg/tags has 81
https://www.thismmalife.com/tags has 78.

I think we should drop the `100` from the title

Reply:
Very good point. I'll fix it. Just gonna have "Top Tags" for now and eventually maybe "Tags"